style.css

```diff
@@ -0,0 +1,85 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;1,100;1,200;1,300;1,400&display=swap");
+
+* {
+  font-family: "Poppins", sans-serif;
+  text-decoration: none;
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+/*
+   dear Roshan Bajgain here I've add the 768px please valid my project because after this I'll be able to move forward
+*/
+
+@media only screen and (max-width: 375px),
+  (max-width: 345px),
+  (max-width: 390px),
+  (max-width: 414px),
+  (max-width: 768px) {
+  body {
```

Comment:
- [ ] Kindly ensure that all the elements have proper alignment, like it is in the figma design.

- Your design
<img width="201" alt="image" src="https://user-images.githubusercontent.com/41163571/234684898-5b83516e-d350-48b2-beb0-8e411178b8ea.png">

- Figma design
<img width="236" alt="right" src="https://user-images.githubusercontent.com/41163571/228056013-322e1f2c-93c1-4f1b-b7e0-46a3b6a945c5.png">